src/styles/table.styl

```diff
@@ -119,6 +120,17 @@
 
 mime-types('fil-file-', '../assets/icons/')
 
+.fil-file-preview
+    content ''
+    position absolute
+    left 0
+    top 50%
+    transform translateY(-50%)
```

Comment:
Oh yeah this technique used to break on older browsers, I honestly don't remember which ones though. That's why I prefer the transform technique as I'm sure it works fine everywhere. But I guess I could give it a try.